Utils/Dataflow/README

```diff
@@ -91,6 +91,30 @@ intended for making sure that information is consistent between ProdSys2 and
 ES. It gets a very basic set of metadata from ProdSys2, adds ES-related
 fields, and checks that it is present in ES rather than uploading it.
 
+================
+* Stages types *
+================
+There are three types of stages:
+
+1. Source stage - an initial stage in a dataflow pipeline that gets data
+from an external source (CDS, Rucio, AMI, etc.)
+
+2. Transform stage - a stage that makes a set of different operations
```

Comment:
1) "... stage that PERFORMS a set of ..." or something else. "Stage makes operations" sounds unnatural.
2) I'm not sure that operations are always different. What if stage performs a single operation? What if it performs several, but all of them are very similar?

Reply:
"... -- these stages perform different operations..."?
The idea, I believe, was to emphasize that "transformation" actions can be really different for different cases. Or maybe not )

Reply:
I still doubt the word "different" here. Also, come to think of it - possibility (and existence of 016) of stage performing a single operation is an argument against "set of" as well, not only "different".
Emphasizing of possible difference - maybe it can be done elsewhere, or in another way?

Reply:
@Evildoor As Marina said, transformation operations can be different, but I can emphasize it like this "...these stages perform a set of operations for different cases". Hope, it sounds correct.

Reply:
Resolving conversation in favor of the next iteration here: https://github.com/PanDAWMS/dkb/pull/278#discussion_r313291818